lib/gamera/page_sections/table.rb

```diff
@@ -188,7 +187,7 @@ def edit_row(name)
       private
 
       attr_reader :headers, :row_css, :row_name, :name_column, :row_class,
-        :row_editor, :row_deleter
+                  :row_editor, :row_deleter
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.